charts/valheim/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+appVersion: latest
+description: Valheim dedicated gameserver with automatic update and world backup support
+name: valheim
+version: 2.0.1
```

Comment:
This should be 1.0.0 right?

Reply:
Ah yes. For context I was testing it out in my own repo and the 2.0.0 was because of switching to using the common template. Starting at 1.0.0 for the k8s-at-home chart makes sense.